### ISSUE TITLE
The «Check out with PayPal» button was replaced by the «Buy Now» button in Magento 2.3.2

### ISSUE THREAD

**Issue by dmitrii-fediuk** [shiftsst/core#10]
## Magento 2.2.8
![02](https://user-images.githubusercontent.com/83204/66011508-43f0a980-e4e5-11e9-8cab-aecf60bbd2a7.png)

## Magento 2.3.2
![01](https://user-images.githubusercontent.com/83204/66011519-4ce17b00-e4e5-11e9-889d-538bed30213f.png)

**Comment by dmitrii-fediuk** [comment]
I have disabled the «**Enable In-Context Checkout Experience**» option
```
bin/magento config:set payment/paypal_express/in_context 0
bin/magento cache:clean
```

[magento.stackexchange.com/a/284753](https://magento.stackexchange.com/a/284753)
[mage2.pro/t/topic/6029](https://mage2.pro/t/topic/6029)

**Comment by dmitrii-fediuk** [comment]
Fixed in [0.0.2](https://github.com/shiftsst/core/tree/0.0.2).
